other_procore_apis/bim_web_viewer.md

```diff
@@ -5,8 +5,45 @@ layout: default
 section_title: Other Procore APIs
 ---
 
+<!-- markdownlint-disable no-inline-html -->
+<style>
+  .header-link-container {
+    padding: 10px 0;
+    position: relative;
+    margin-top: -52px;
+    height: 32px;
+  }
+  .header-link-container:hover > .header-link {
+    opacity: 1;
+  }
+  .header-link {
+    font-size: 12px;
+    padding: 5px;
+    opacity: 0;
+    position: absolute;
+  }
+  .header-link::before {
+    content: '\1f517'; /*link character*/
+  }
+  .header-link:hover {
+    opacity: 1;
+  }
+
+  h2 + .header-link-container > .header-link {
+    transform: translate(-26px, 2px);
+  }
+  h3 + .header-link-container > .header-link {
+    transform: translate(-26px, 0px);
+  }
+  h4 + .header-link-container > .header-link {
+    transform: translate(-26px, -2px);
+  }
+</style>
+
 ## Getting started
 
+<p class="header-link-container"><a class="header-link" href="#getting-started"></a></p>
```

Comment:
Using a `p` tag specifically here (instead of a `div`) because markdown will wrap every block of text in a `p` tag thinking that it is content. If we add our own `p` tag it seems to not do that.